framework/test/bundles/libC1/TestBundleC1.cpp

```diff
@@ -132,7 +136,8 @@ namespace cppmicroservices
                         count--;
                     }
                 }
-                // stop spin
+                // Rather than a condition variable, we are looping until regs is non-empty.
+                // This sleep stops the code from looping too quickly and causing slow downs.
```

Comment:
Noting that we are choosing to sleep (which is more inefficient) instead of using a `condition_variable` because of a false positive coming from tsan when using `wait_for` would be useful.